app/build.gradle

```diff
@@ -98,7 +98,7 @@ dependencies {
   // Google Play Services availability check
   implementation 'com.google.android.gms:play-services-base:15.0.1'
```

Comment:
Shouldn't we update this dependency too?

Reply:
15.0.1 is the newest version